content/en/docs/reference/setup-tools/kubeadm/kubeadm-init.md

```diff
@@ -142,6 +142,22 @@ For information about enabling IPVS mode with kubeadm see:
 For information about passing flags to control plane components see:
 - [control-plane-flags](/docs/setup/production-environment/tools/kubeadm/control-plane-flags/)
 
+### Running kubeadm without an internet connection {#without-internet-connection}
+
+For running kubeadm without an internet connection you have to pre-pull the required control-plane images.
```

Comment:
Please write 'Internet' rather than 'internet'. Strictly speaking, these two words represent different things.

Reply:
changing to "Internet" regardless.